src/CarbonAware.CLI/test/integrationTests/Commands/Location/LocationCommandTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using CarbonAware.DataSources.Configuration;
+using NUnit.Framework;
+using System.Text.Json;
+
+namespace CarbonAware.CLI.IntegrationTests.Commands.Location;
+
+/// <summary>
+/// Tests that the CLI handles and packages various responses from aggregators 
```

Comment:
This comment seems off for these tests, think you mean to have something about the location commands